lib/modules/manager/tflint-plugin/readme.md

```diff
@@ -0,0 +1 @@
+This will maintain the [TFLint configuration file](https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/config.md), and renovates the plugins it contains.
```

Comment:
```suggestion
Renovate maintains your [TFLint configuration file](https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/config.md), and updates the plugins in the file.
```

How about this?

I don't like the non-standard verb `renovates`, we should use `updates` instead as that is a word every software engineer understands. 😉

Reply:
Agree, I picked up that wording from another readme but I also found it weird :)

Reply:
> I picked up that wording from another readme but I also found it weird :)

@multani Can you give me a link to that other readme?